incidents_mttr_example.py

```diff
@@ -7,45 +7,58 @@
 # Version:      1.0
 #
 
-import json
 import argparse
 from datetime import date, datetime, timedelta
 import csv
 import demisto
 
+
 def format_dt(dt):
     return dt.strftime('%Y-%m-%dT%H:%M:%S')
 
+
 def parse_dt(dt):
     return datetime.strptime(dt[:19], '%Y-%m-%dT%H:%M:%S')
 
+
 def p(what):
     if verbose:
         print(what)
 
+
 def options_handler():
-    parser = argparse.ArgumentParser(description='Incident MTTR statistics for a time period')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
+    parser = argparse.ArgumentParser(
+        description='Incident MTTR statistics for a time period')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
```

Comment:
> Note: Required options are generally considered bad form because users expect options to be optional, and thus they should be avoided when possible.

* https://docs.python.org/3/library/argparse.html#required

Maybe these two should be positional arguments instead?

```python
parser.add_argument('key', help='The API key to access the server')
parser.add_argument('server', help='The server URL to connect to')

```

Reply:
used positional arguments instead of required=True.(e2beb08)